src/components/UserManagement/SetUpFinalDayButton.jsx

```diff
@@ -25,8 +25,10 @@ const SetUpFinalDayButton = (props) => {
     if (isSet) {
       updateUserFinalDayStatus(props.userProfile,"Active",undefined)(dispatch)
       setIsSet(!isSet);
-      toast.success("This user's final day has been set.");
-      window.location.reload();      
+      toast.success("This user's final day has been delete.");
```

Comment:
deleted?

Reply:
Sure, will revise here. Is there anything else that I need to revise?